website/static/js/myProjects.js

```diff
@@ -268,18 +268,22 @@ function _formatDataforPO(item) {
     if (contributorsData){
         item.embeds.contributors.data.forEach(function(c){
             var attr;
-            var users = lodashGet(c, 'embeds.users.data', null);
-            if (users) {
-                if (users.errors) {
-                    attr = users.errors[0].meta;
-                } else {
-                    attr = users.attributes;
-                }
+
+            if (lodashGet(c, 'attributes.unregistered_contributor', null)) {
+                item.contributors += c.attributes.unregistered_contributor;
             }
-            if (attr) {
-                item.contributors += attr.full_name + ' ' + attr.middle_names + ' ' + attr.given_name + ' ' + attr.family_name + ' ' ;
+            else {
+                var users = lodashGet(c, 'embeds.users.data', null);
+                if (users) {
+                    if (users.errors) {
+                        attr = users.errors[0].meta;
+                    } else {
+                        attr = users.attributes;
+                    }
+                }
+                if (attr)
+                    item.contributors += attr.full_name + ' ' + attr.middle_names + ' ' + attr.given_name + ' ' + attr.family_name + ' ' ;
```

Comment:
Likely outside the scope of this PR but I'm noticing some duplication between this code and project-organizer.js, in particular for mapping API responses into a consistent User interface. It might be beneficial to encapsulate this as JS class to keep all of this logic in once place.